internal/discord/commands.go

```diff
@@ -112,7 +112,98 @@ func (d *Discord) Purge(s *discordgo.Session, i *discordgo.InteractionCreate) {
 //	duration:	string
 //	reason:		string
 func (d *Discord) Exile(s *discordgo.Session, i *discordgo.InteractionCreate) {
-	return
+	optionMap := mapOptions(i)
+	logMsg, err := d.LogCommand(i.Interaction)
+	if err != nil {
+		fmt.Printf("Failed to log: %v\n", err)
+	}
+
+	state := &InteractionState{
+		session:     s,
+		interaction: i,
+		logMsg:      logMsg,
+		isFirst:     true,
+	}
+
+	// Calculate duration of exile
+	startTime := time.Now()
+	duration, err := parseDuration(optionMap["duration"].StringValue())
+	if err != nil {
+		fmt.Printf("Failed to parse duration: %v\n", err)
+		return
+	}
+
+	userToExile := optionMap["user"].UserValue(nil)
+
+	// Check if user meets the requirements for an exile
+	member, err := d.ExileCheckUserHelper(state, userToExile.ID)
+	if err != nil {
+		return
+	}
```

Comment:
ehhhh if we're already error checking in the helper funcs we might as well get rid of it here